Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -265,6 +265,127 @@ using TrackExtra = TracksExtra::iterator;
 using FullTracks = soa::Join<Tracks, TracksCov, TracksExtra>;
 using FullTrack = FullTracks::iterator;
 
+namespace trackfwd
+{
+// TRACKPARFWD TABLE definition
+DECLARE_SOA_INDEX_COLUMN(Collision, collision);
+DECLARE_SOA_COLUMN(TrackType, trackType, uint8_t); // TODO change to ForwardTrackTypeEnum when enums are supported
+DECLARE_SOA_COLUMN(X, x, float);
+DECLARE_SOA_COLUMN(Y, y, float);
+DECLARE_SOA_COLUMN(Z, z, float);
+DECLARE_SOA_COLUMN(Phi, phi, float);
+DECLARE_SOA_COLUMN(Tgl, tgl, float);
+DECLARE_SOA_COLUMN(Signed1Pt, signed1Pt, float);
+
+DECLARE_SOA_DYNAMIC_COLUMN(Charge, charge, [](float signed1Pt) -> short { return (signed1Pt > 0) ? 1 : -1; });
```

Comment:
Somewhat off-topic: this way of extracting the charge from ``q/pt``, inherited from the barrel tracks (in turn, coming from Aliroot), does not allow proper treatment of charges other than +/- 1. For the forward tracks this might be not a problem, since there we will not consider (is it true also for MFT?) charge 2 tracks, and for B=0 the ``q/pt`` is anyway 0. 
But for the barrel we will have both charge 2 particles and charge 0 particles with measured momentum (V0s).
For this reason I've added https://github.com/AliceO2Group/AliceO2/blob/f14182b852daaa405170019c0b9a7c96830dec5e/DataFormats/Reconstruction/include/ReconstructionDataFormats/TrackParametrization.h#L231 which also should be propagated to AOD.
Perhaps it is worth to introduce it also for forward tracks.
Another variable which might be worth propagating (for barrel tracks only) is the PID assigned during the tracking: https://github.com/AliceO2Group/AliceO2/blob/f14182b852daaa405170019c0b9a7c96830dec5e/DataFormats/Reconstruction/include/ReconstructionDataFormats/TrackParametrization.h#L232, which proved to be useful in understanding problems in the data.

Reply:
Thanks for your feedback. Let's bring your remarks up briefly next week at the WP14 meeting. Here some feedback from my side:
Charge:
* The V0s go to a different table, so no worry about that
* Would you assign charge=2 tracks clearly at the reconstruction? Otherwise it would be anyway analysis level
PID hypothesis:
* This looks like a debug / QA information to me as you write also. We could consider adding an extra QA table to the data model which we stop filling at some point. I would refrain from adding it to the standard track table.

Reply:
I think forward tracks should be prepared for charges other than +-1, but I rather let this for a forthcoming revision of the forward track classes. 
I am adding a new commit naming MUONStandalone track, but will keep the pDCA calculation as it is so that any tests performed by @vfeuilla gives identical results following the migration from muon -> trackfwd.

I think this PR is OK to go.

Reply:
I think before merging we should conclude on the discussion about pDca etc on the fly versus storing.
If @vfeuilla wants an exact comparison one can always do a calculation in the analysis task. And this should anyway happen locally. No need to have everything in the dev branch. @vfeuilla can just use your branch for the checks.

So the questions remaining are:
* If we can do on the fly for all but MUONStandalone. How do we split the two cases, and is it worth it.
* What is the expected share of those tracks?
* You also mentioned indices that refer between the tracks where are they. Note that the framework does not yet support indices to the same table, you have to declare them as int for now
* Could you add a comment after each stored member to explain the comment (we will also do that for the rest of the data model)?
* I think there are some unanswered questions from Ruben about the `z` here...

Reply:
About splitting the cases that need storage from those who do not, perhaps we could implement conditional columns, or have a separate table for MUONStandalone's PDca and RAtAbsorberEnd?

Rought estimates of the shares:

* MUONStandalone is the smallest share of forward tracks (about 4 per PbPb event), which needs PDca and storage. 
* MFTStandalone has the highest storage share, order of 1000 per PbPB event - PDca can be evaluated on the fly.
* GlobalMuonTracks share not know yet. May be about 10 times the MUONStandalone share.  - PDca can be evaluated on the fly.

As for the Z component, I do not know why it is accounted here. O copied the columns used by @vfeuilla tests for a smooth transition from the muon->trackfwd table. @iarsene and @vfeuilla: are you OK in dropping the EXPRESSION_COLUMNs for PDca and RAtAbsorberEnd in favor of the SOA_COLUMNs?